data/campaigns/World_Conquest/Modding.md

```diff
@@ -0,0 +1,490 @@
+        # recruit list. For your own benefit, write the recruit list with pairs side by side
+        #as not to be confused later on
+        image= # image of the faction
+        type=random
+        leader= Dwarvish Steelclad,Dwarvish Thunderguard,Dwarvish Stalwart,Rogue,Trapper,Drake Flare,Fire Drake,Drake Arbiter,Drake Thrasher,Drake Warrior,Saurian Oracle,Saurian Soothsayer
+        random_leader= # list of leader ids which should be chosen if randomised
+        description= # description of the faction (you can skip this one)
+        [world_conquest_data]
+            commanders=Drakes,Knalgans
+            heroes=Loyalists_All,Northerners_All,Young Ogre
+            deserters=Rebels,Undead,Dune Burner
+            deserters_names={WC_STR_DUNE_BURNER}+", "+{WC_STR_REBELS}+{WC_STR_AND}+{WC_STR_THE_UNDEAD}
+            {WC_II_PAIR "Drake Fighter" "Dwarvish Fighter"}
+            {WC_II_PAIR "Drake Burner" "Dwarvish Thunderer"}
+            {WC_II_PAIR "Saurian Augur" "Dwarvish Ulfserker"}
+            {WC_II_PAIR "Drake Clasher" "Dwarvish Guardsman"}
+            {WC_II_PAIR "Saurian Skirmisher" "Poacher"}
+            {WC_II_PAIR "Drake Glider" "Footpad"}
+            {WC_II_PAIR "Thief" "Thief"}
+        [/world_conquest_data]
+    [/multiplayer_side]
+
+
+    {QUICK_4MP_LEADERS}
+    {TURNS_OVER_ADVANTAGE}
+
+    # load the main resource
+    # otherwise you will encounter errors
+    [load_resource]
+        id = extended_era_resource_main
+    [/load_resource]
+[/era]
+#endif
+```
+
+## How to make your own artifacts and have them added to World Conquest
+
+```ini
+[era]
+    # era code here
+
+    [world_conquest_data]
+        [artifact]
+            name= _ "Leather Bag of Herbal Stuff"
+            icon=items/leather-pack.png # dont add attack icons here, just what the item image here.
+            description= _ "heals +4 and regenerates +4"
+            info = _ "Some story about random Dune Alchemist dude deciding to pack his stuff in a bag and it got lost."
+            sound=heal.wav
+            [effect]
+                apply_to=remove_ability
+                [abilities]
+                    [heals]
+                        id=healing
+                    [/heals]
+                [/abilities]
+            [/effect]
+            [effect]
+                apply_to=new_ability
+                [abilities]
+                    {ABILITY_HEALS}
+                    {ABILITY_SELF_HEAL}
+                [/abilities]
+            [/effect]
+        [/artifact]
+    [/world_conquest_data]
+[/era]
+```
+
+## How to add your own trainers
+
+```ini
+[era]
+    # era code here
+
+    [world_conquest_data]
+        [trainer]
+            type=Blood Manipulator
+            advanced_type=Sangel
+            image=attacks/wail.png
+            name= _ "Blood Magic"
+            dialogue= "You have found me, mortals? Well...let us show your recruits some blood magic!"
+            [grade]
```

Comment:
A comment about what each 'grade' is might be helpful (I'm not a WC player, maybe it would be obvious to anyone who is writing such an add-on).